CHANGELOG.md

```diff
@@ -5,6 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.2]
```

Comment:
Minor version can be update : `0.7.0`, this is more than a patch.  In semver, you can increment `MINOR version when you add functionality in a backward compatible manner`

Reply:
Agree. Changed.